homeassistant/components/iammeter/sensor.py

```diff
@@ -40,6 +58,51 @@
 PLATFORM_TIMEOUT = 8
 
 
+def _migrate_to_new_unique_id(
```

Comment:
Entity registry migrations should always be accompanied with tests. It's easy to get something wrong when doing migrations.